Remotion/Web/Development.WebTesting/Utilities/RetryUntilTimeout.cs

```diff
@@ -55,27 +56,58 @@ public void Run ()
       _retryUntilTimeout.Run();
     }
 
-    public static void Run ([NotNull] Action action)
+    /// <summary>
+    /// Executes a given <see cref="Action"/> repeatedly (using the given retry interval) until no exception is thrown during execution or until the
+    /// given timeout has been reached (in which case the final exception is rethrown).
+    /// </summary>
+    /// <param name="action">The <see cref="Action"/> to be executed repeatedly.</param>
+    /// <param name="timeout">
+    /// The timeout after which no more retries are made and the final exception is rethrown. Can be <see langword="null" />. If <see langword="null" />,

```

Comment:
...rethrown. Defaults to <see cref="DriverConfiguration.SearchTimeout"/> if no value is provided.